### PR TITLE
Link incident response form

### LINKED ISSUE #96: Add anonymous report form
* Make a hackuci.com/report route that redirects to our anonymous report form
* Sydnee will provide later today

### PR DESCRIPTION
**Pull request by samderanova**
- Add link to form in navbar
- Create new component that redirects to the form when accessing https://hackuci.com/report

### CODE REVIEW

**Review by taesungh — CHANGES_REQUESTED** [review]
We can use [Next.js Rewrites](https://nextjs.org/docs/api-reference/next.config.js/rewrites) for this, have a look at 1dd5839.

**Review by rileysw — DISMISSED** [review]

File: src/components/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -55,6 +55,9 @@ function Navigation() {
 						<PrivateNavLinkItem authorized={isAdmin} href="/admin/dashboard">
 							Admin
 						</PrivateNavLinkItem>
+						<NavLinkItem href="https://forms.gle/yM8Revi2NLHGNKs17">
```
Comment (taesungh) [inline src/components/Navigation/Navigation.tsx:58]: And we could now link the source URL from the navigation i.e. `/report` instead of the form.

Comment (samderanova) [reply src/components/Navigation/Navigation.tsx]: Updated in `532f374`

**Review by rileysw — APPROVED** [review]

**Review by taesungh — APPROVED** [review]
Nice and simple, thanks for the quick turnaround. Will be squashing during merge.

**Pull request merged**